CMake/External_OpenBLAS.cmake

```diff
@@ -44,11 +44,20 @@ endif()
 
 fletch_external_project_force_install(PACKAGE OpenBLAS)
 
+
 set(OpenBLAS_ROOT ${fletch_BUILD_INSTALL_PREFIX} CACHE STRING "")
 
+# Find the OpenBLAS library name that will be passed to OpenCV / Caffe / etc..
+# Note: the lib does not exist yet, but we can be pretty sure what it will be
+get_system_library_name(openblas openblas_libname)
+set(OpenBLAS_LIB "${OpenBLAS_ROOT}/lib/${openblas_libname}")
+set(OpenBLAS_INCLUDE_DIR "${OpenBLAS_ROOT}/include")
+
 file(APPEND ${fletch_CONFIG_INPUT} "
 ########################################
 # OpenBLAS
 ########################################
 set(OpenBLAS_ROOT    \${fletch_ROOT})
+set(OpenBLAS_LIB     \${OpenBLAS_LIB})
+set(OpenBLAS_INCLUDE_DIR \${OpenBLAS_INCLUDE_DIR})
```

Comment:
@dstoup I'm unsure if these should go here.

Reply:
I think this is the right location for these variables.

Reply:
Other packages don't seem to output these variables in the `fletch_CONFIG_INPUT`. Is it the case that the other packages should be doing this? Or are packages free to export however much they want?

Reply:
I was having this conversation with someone recently. I think it makes like easier if all of the knowledge about a package is generated in its configuration. That eliminates the duplication of get_library_name type stuff.